testdata/p4_16_samples_outputs/annotations-first.p4

```diff
@@ -1,6 +1,7 @@
 @annotest const bit<1> b = 1w1;
 extern Annotated {
     Annotated();
+    void execute(bit<8> index);
```

Comment:
looks like ToP4 is failing to output the annotations on the methods?  Or is something deleting them?

Reply:
In this case you may have to do some more work in toP4.cpp